reference/system-databases/cluster-load.md

```diff
@@ -0,0 +1,65 @@
+---
+title: CLUSTER_LOAD
+summary: Learn the `CLUSTER_LOAD` cluster load table.
+category: reference
+---
+
+# CLUSTER_LOAD
+
+The `CLUSTER_LOAD` cluster load table provides the current load information of the server where each node of the TiDB cluster is located.
+
+{{< copyable "sql" >}}
+
+```sql
+desc cluster_load;
+```
+
+```
++-------------+--------------+------+------+---------+-------+
+| Field       | Type         | Null | Key  | Default | Extra |
++-------------+--------------+------+------+---------+-------+
+| TYPE        | varchar(64)  | YES  |      | NULL    |       |
+| INSTANCE    | varchar(64)  | YES  |      | NULL    |       |
+| DEVICE_TYPE | varchar(64)  | YES  |      | NULL    |       |
+| DEVICE_NAME | varchar(64)  | YES  |      | NULL    |       |
+| NAME        | varchar(256) | YES  |      | NULL    |       |
+| VALUE       | varchar(128) | YES  |      | NULL    |       |
++-------------+--------------+------+------+---------+-------+
+```
+
+Field description:
+
+* `TYPE`: Corresponds to the `TYPE` field in the [`information_schema.cluster_info`](/reference/system-databases/cluster-info.md) table. The optional values are `tidb`, `pd`, and `tikv`.
+* `INSTANCE`: Corresponds to the `STATUS_ADDRESS` field in the [`information_schema.cluster_info`](/reference/system-databases/cluster-info.md) cluster information table.
```

Comment:
ditto

Reply:
@reafans I see that [the Chinese version](https://pingcap.com/docs-cn/dev/reference/system-databases/cluster-load/) still uses `STATUS_ADDRESS` in `cluster-load.md`. Would you please check and update all necessary places?

Reply:
We have tested some place in sql-diagnosis and fixed some unreasonable detail recently. And i will check and update the Chinese version recently。